jung-visualization/src/main/java/edu/uci/ics/jung/visualization/subLayout/TreeCollapser.java

```diff
@@ -49,6 +52,29 @@ public static void expand(MutableCTreeNetwork tree, CTreeNetwork subTree) {
             ? tree.inEdges(subTree).iterator().next() // THERE CAN BE ONLY ONE
             : null;
     tree.removeNode(subTree);
-    TreeUtils.addSubTree(tree, subTree, parent.orElse(null), parentEdge);
+    if (parent.isPresent()) {
```

Comment:
Consider adding something prior to this if/else that simply returns if subTree() is empty.  Then you don't have to do a check inside copyInto.

Reply:
Although whether that makes sense will depend on whether copyInto becomes its own thing (see below).

Reply:
Thanks for the feedback! I've changed things as as I believe you've suggested, but it's not really clear to me that it's much of an improvement. WDYT?

Reply:
Oh, I only just saw your comment about whether `copyInto` will become it's own thing! I'll look at the comments that it points to in a moment. :)

Reply:
I've gone and removed `copyInto`, as it looks to me that it's no longer needed. :)